server.go

```diff
@@ -1381,6 +1381,7 @@ func (sp *serverPeer) OnBlock(_ *peer.Peer, msg *wire.MsgBlock, buf []byte) {
 func (sp *serverPeer) OnInv(_ *peer.Peer, msg *wire.MsgInv) {
 	// Ban peers sending empty inventory announcements.
 	if len(msg.InvList) == 0 {
+		peerLog.Errorf("%s: empty inventory announcement", sp)
```

Comment:
`Error` implies the user can/needs to do something about it.  This is not the case.

Moreover, it is crypto to an ordinary user.  This is 100% guaranteed to generate nothing but support questions as it stands.

I'm fine with showing something, but it should be `Warnf` to match what `addBanScore` does where it takes a reason and it should explain what is happening.

That is:

```suggestion
		peerLog.Warnf("Misbehaving peer %s: empty inventory announcement -- "+
			"banning and disconnecting", sp)
sp)
```